extensions/interactions/CodeRepl/directives/CodeRepl.js

```diff
@@ -29,7 +29,8 @@ oppia.directive('oppiaInteractiveCodeRepl', [
       restrict: 'E',
       scope: {
         onSubmit: '&',
-        getLastAnswer: '&lastAnswer'
+        getLastAnswer: '&lastAnswer',
+        setAnswerValidity: '&' // Must call whenever the answer changes.
```

Comment:
Couple of nits (sorry):
- We normally always have comments on top of the line they refer to.
- Can we say `This should be called whenever the answer changes.`? I think the current formulation is slightly unclear; my first thought was "what does this thing need to call?"

Reply:
Got it.